src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -649,35 +640,6 @@ private function addRouterSection(ArrayNodeDefinition $rootNode): void
     private function addSessionSection(ArrayNodeDefinition $rootNode): void
     {
         $rootNode
-            ->validate()
-                ->always(function (array $v): array {
-                    if ($v['session']['enabled']) {
-                        if (!\array_key_exists('cookie_secure', $v['session'])) {
-                            trigger_deprecation('symfony/framework-bundle', '6.4', 'Not setting the "framework.session.cookie_secure" config option is deprecated. It will default to "auto" in 7.0.');
-                        }
-
-                        if (!\array_key_exists('cookie_samesite', $v['session'])) {
-                            trigger_deprecation('symfony/framework-bundle', '6.4', 'Not setting the "framework.session.cookie_samesite" config option is deprecated. It will default to "lax" in 7.0.');
-                        }
-
-                        if (!\array_key_exists('handler_id', $v['session'])) {
-                            if (!\array_key_exists('save_path', $v['session'])) {
-                                trigger_deprecation('symfony/framework-bundle', '6.4', 'Not setting the "framework.session.save_path" config option when the "framework.session.handler_id" config option is not set either is deprecated. Both options will default to "null" in 7.0.');
-                            } else {
-                                trigger_deprecation('symfony/framework-bundle', '6.4', 'Not setting the "framework.session.handler_id" config option is deprecated. It will default to "session.handler.native_file" when "framework.session.save_path" is set or "null" otherwise in 7.0.');
-                            }
-                        }
-                    }
-
-                    $v['session'] += [
-                        'cookie_samesite' => null,
-                        'handler_id' => 'session.handler.native_file',
-                        'save_path' => '%kernel.cache_dir%/sessions',
```

Comment:
when `handler_id` is non-null, save_path should still default to `%kernel.cache_dir%/sessions`, how can we deal with that?